### PR TITLE
mozbackup: Add version 1.5.1

### PR DESCRIPTION
**Pull request by owendaprile**
Add [MozBackup](http://mozbackup.jasnapaka.com/).
It backs up profiles from Mozilla programs.

### CODE REVIEW

**Comment by linsui** [comment]
It's open source, see http://mozbackup.jasnapaka.com/development.php . Would it be better to download from sourceforge for https?

**Comment by owendaprile** [comment]
Like @linsui said, it's licensed under the three licenses I originally had. 
I'm not really sure what the extra files it has are, as it doesn't have any settings exposed in the GUI and does it's behavior doesn't change on subsequent runs.

**Comment by Ash258** [comment]
Source code is licensed under all of that, but artifacts are freeware as stated on homepage.

> This program is freeware (even for commercial use) and works on Windows

See vscode as other example. Code is licensed under [MIT](https://github.com/microsoft/vscode/blob/master/LICENSE.txt), but final software is under [freeware proprietary license](https://code.visualstudio.com/License/)

File: bucket/mozbackup.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{
+    "version": "1.5.1",
+    "description": "Create backups of profiles from Mozilla software.",
+    "homepage": "http://mozbackup.jasnapaka.com/",
```
Comment (Ash258) [inline bucket/mozbackup.json]:
```suggestion
    "homepage": "http://mozbackup.jasnapaka.com",
```

**Review by Ash258 — APPROVED** [review]

**Pull request merged**